.github/workflows/haskell.yml

```diff
@@ -58,42 +58,69 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        name:
+          # - 9.2.1
```

Comment:
Is there some issue with 9.2.1?

Reply:
No, this is just a CI file revision. I'll add in 9.2.1